datastore/importers/nmdb-import-show-neighbor/Parser.cpp

```diff
@@ -199,6 +200,45 @@ Parser::Parser() : Parser::base_type(start)
        pnx::bind(&nmdo::InterfaceNetwork::addReachableMac, &qi::_val, qi::_b))]
     ;
 
+  // =============================================================================
+  // ARP: Juniper Config
+  // =============================================================================
+
+  arpJuniperConfig =
+    arpHeaderJuniper >>
```

Comment:
I realize the rest of the parser's logic doesn't do what I'm about to state (since it was made prior to the decision), but parser logic should only use the sequence operator, `>>`, when the ability to backtrack is explicitly wanted.  Prefer to use the expectation operator, `>`, in most cases as it is, overall, more explicit and efficient for our purposes.  As such, the majority of this new code should be switched.